pipeline-runner/R/gem2s-7-upload_to_aws.R

```diff
@@ -24,17 +25,19 @@ upload_to_aws <- function(input, pipeline_config, prev_out) {
     key = experiment_id
   )
 
-  # seurat object to s3
-  message("Uploading Seurat Object to S3 ...")
-  fpath <- file.path(tempdir(), "experiment.rds")
-  saveRDS(scdata, fpath, compress = FALSE)
+ for (sample in names(scdata_list)) {
+    message("Uploading sample ", sample, " object to S3 ...")
+    fpath <- file.path(tempdir(), "experiment.rds")
+    saveRDS(scdata_list[[sample]], fpath, compress = FALSE)
+    # can only upload up to 50Gb because part numbers can be any number from 1 to 10,000, inclusive.
```

Comment:
I don't understand this comment. Can you explain?

Reply:
explained